doc/source/development.rst

```diff
@@ -1,264 +1,77 @@
 
-You should also be familiar with our |Jenkins| setup.
+We keep trunk in a constant state of stability to allow for a release at
+any time and to minimize wasted time by developers due to broken code.
 
 Overview
-~~~~~~~~
-At Percona we use `Bazaar <http://www.bazaar-vcs.org>`_ for source
-control and `launchpad <http://www.launchpad.net>`_ for both
-code hosting and release management.
-
-Changes to our software projects could be because of a new feature
-(blueprint) or fixing a bug (bug). Projects such as refactoring could
-be classed as a blueprint or a bug depending on the scope of the work.
-
-Blueprints and bugs are targeted to specific milestones (releases). A
-milestone is part of a series - e.g. 1.6 is a series in Percona
-XtraBackup and 1.6.1, 1.6.2 and 1.6.3 are milestones in the 1.6 series.
-
-Code is proposed for merging in the form of merge requests on launchpad.
-
-Some software (such as Percona Xtrabackup) we maintain both a
-development branch and a stable branch. For example: Xtrabackup 1.6 is
-the current stable series, and changes that should make it into bugfix
-releases of 1.6 should be proposed for the 1.6 tree. However, most new
-features or more invasive (or smaller) bug fixes should be targeted to
-the next release, currently 1.7. If submitting something to 1.6, you
-should also propose a branch that has these changes merged to the
-development release (1.7). This way somebody else doesn't have to
-attempt to merge your code and we get to run any extra tests that may
-be in the tree (and check compatibility with all platforms).
-
-For Percona Server, we have two current bzr branches on which
-development occurs: 5.1 and 5.5. As Percona Server is not a
-traditional project, instead being a set of patches against an
-existing product, these two branches are not related. That is, we do
-not merge from one to the other. To have your changes in both, you
-must propose two branches: one for 5.1 version of patch and one for
-5.5.
-
-Making a change to a project
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-In this case we're going to use percona-xtrabackup as an
-example. workflow is similar for Percona Server, but patch will need
-to be modified both in 5.1 and 5.5 branches.
-
-* ``bzr branch lp:percona-xtrabackup featureX`` (where 'featureX' is a
+--------------------------------------------------------------------------------
+
+At Percona we use `Git <https://git-scm.com>`_ for source control, `GitHub
+<https://github.com/percona>`_ for code hosting, and `Jira
+<https://jira.percona.com>`_ for release management.
+
+We change our software to implement new features or to fix bugs. Refactoring
+could be classed either as a new feature or a bug depending on the scope of
+work.
+
+New features and bugs are targeted to specific milestones (releases). A
+milestone is part of a series. For example, 2.4 is a series in Percona
+XtraBackup and 2.4.15, 2.4.16 and 2.4.17 are milestones in this series.
```

Comment:
We can eliminate the term milestone and just use release.  Milestone is actually a leftover term from Launchpad where they called releases milestones.

Reply:
Got it. Will fix.